posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+the parent folder `two_fovs` (@fig-stitch-widget2). Your slide name will be labeled something else. 
+Once you select `Open`, you should see the path to the raw data folder. If an unexpected format
+was detected, there will be an error message (_e.g._, @fig-stitch-widget3).
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-stitch-widget2"
+#| fig-cap: "Example showing a launched `napari-cosmx` plugin with a possibly-unrelated slide. To stitch, a new slide, Click `Browse...` in the `Stitch Images` widget located on the right-hand side of napari."
+
+knitr::include_graphics("./figures/stitch-widget2.png")
+```
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-stitch-widget3"
+#| fig-cap: "Stitching widget prints the path to the correctly formatted raw data (left) or provides an error message if not formatted correctly (right). Note that only the correctly formatted location can proceed to the next step (choosing otuput folder)."
+
+knitr::include_graphics("./figures/stitch-widget3.png")
+```
+
+Next, select the location where you want the plugin to return the napari-ready files.
+It is recommend not to store it in the same location as the raw data. Here, I'm 
+pointing it to a location adjacent to the raw data that I named `two_fovs_napari_files`
+(@fig-stitch-widget4).
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-stitch-widget4"
+#| fig-cap: "Select output folder."
+
+knitr::include_graphics("./figures/stitch-widget4.png")
+```
+
+Finally, click `Stitch`. Note: currently there is no refreshing or printing of
+messages. Please do not click `Stitch` more than once. You may see napari become
+unresponse, see the "spinning beachball" (Mac), _etc._ Depending on the number 
+of FOVs, computer configuration, and analyte type, this can take several minutes. 
+Once complete, you should
+see messages that resemble that of @fig-stitch-widget5. If you see the last line
+`See output folder for results`, you successfully converted the raw data into napari-ready files!
+
+To view the results, simply close napari, reopen it, and drag your newly created results
+into the application. 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-stitch-widget5"
+#| fig-cap: "Messages from a successfully completed stitching run."
+
+knitr::include_graphics("./figures/stitch-widget5.png")
+```
+
+
+
+## Adding and viewing metadata {#sec-adding-metadata} 
+
+While we'll have dedicated posts that discuss tips and tricks for using the
+`napari-cosmx` plugin, here we'll discuss one of the most powerful uses: viewing 
+cells types. 
+
+While not needed for the basic stitching, the Seurat file that is downloaded
+from AtoMx can containing important cell-level information. For example, 
+if cell typing was performed in AtoMx, each cell will have a label with it's cell
+type. 
+
+In this section, I'll show you the basic principle for converting the meta data
+within the Seurat object in to a csv file that can be understood by `napari-cosmx`. 
```

Comment:
in to -> into

Reply:
Done: 208857d88eb7bd81da7fd043536a8aad6f51e28b